src/main/java/nz/co/blink/debit/client/v1/QuickPaymentsApiClient.java

```diff
@@ -443,6 +317,7 @@ private WebClient.Builder getWebClientBuilder(String requestId) {
 
         return WebClient.builder()
                 .clientConnector(connector)
+                .defaultHeader(HttpHeaders.USER_AGENT, "Java/Blink SDK 1.0")
```

Comment:
since the "Java/Blink SDK 1.0" user agent string is used in multiple places, we should create a constant for it somewhere